Makefile.pkg

```diff
@@ -0,0 +1,66 @@
+# This Makefile contains RPM related targets
+COMMIT=$(shell git log --pretty=format:'%H' -n 1)
+COMMIT_DATE=$(shell git log --pretty='format:%cd' --date='format:%Y%m%d' -n 1)
+SHORT_COMMIT=$(shell git rev-parse --short=9 HEAD)
+MOCK_CONFIG=default
+ARCHIVE_BASE_NAME=avocado
+PYTHON_MODULE_NAME=avocado-framework
+RPM_BASE_NAME=python-avocado
+ifndef VERSION
+VERSION=$(shell $(PYTHON) setup.py --version 2>/dev/null)
+endif
+
+all:
+	@echo
+	@echo "RPM related targets:"
+	@echo "source: Create single source package with commit info, suitable for RPMs"
+	@echo "srpm:   Generate a source RPM package (.srpm)"
+	@echo "rpm:    Generate binary RPMs"
+	@echo
+	@echo "Release related targets:"
+	@echo "source-release:  Create source package for the latest tagged release"
+	@echo "srpm-release:    Generate a source RPM package (.srpm) for the latest tagged release"
+	@echo "rpm-release:        Generate binary RPMs for the latest tagged release"
+	@echo
+	@echo "variables:         Show the value of variables as defined in this Makefile or"
+	@echo "                   given as input to make"
+	@echo
+
+clean:
+	rm -rf SOURCES BUILD
```

Comment:
When using the default Makefile, calling `make clean` it will also remove `SOURCES` and `BUILD` directories. So, to avoid any kind of contamination I'm wondering if it wouldn't be safer to call `make clean` here. What do you think?

Also, would be nice to have a help message for clean as well, specially if there is a difference between this and the main one.

Reply:
First, This makefile shouldn't depend on the main one at all. It's a self-contained one for packaging  stuff.
Second, I was in fact planning to modify the clean command in setuptools to not longer remove SOURCES and BUILD, since it doesn't make sense anymore to remove it there. 
Ok for the help message.

Reply:
Yes, makes sense. Thanks.